### ISSUE TITLE
VS 16.10 Bug - Creates Empty Solution

### ISSUE THREAD

**Issue by RehanSaeed** [sayedihashimi/template-sample#29]
When I run the following command, everything works:

```
dotnet new --install Boxed.Templates
dotnet new api
```

If I select the same options in VS 16.10's new UI, VS opens a blank solution and no files are output to disk except the solution file. How can I debug what went wrong?

Is this the right place for this bug or is it dotnet/templating?

**Comment by sayedihashimi** [comment]
Do you have any required parameters that don't have a default value?

**Comment by lohithgn** [comment]
@sayedihashimi  @RehanSaeed 

I just tried this with VS Community 2019 Preview V16.11.0 Preview 1.0.  I used Web API template. 

@sayedihashimi question intrigued me. The template did not work for me the first time I tried. First time I tried - I had entered everything except Application Insight field's value.  I tried second time and this time I entered a dummy value to application insights. And I can see the project created successfully.

As consumers - we will need to make sure that we enter all required values as @sayedihashimi indicates. or the template needs to put a default value for required params.

Hope this helps.
